### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by ErwanAliasr1**
This PR is about improving the actual documentation

### CODE REVIEW

**Comment by ErwanAliasr1** [comment]
@fredericlepied Updated the basename as we discussed it yesterday. Minimal documentation update too.

File: pxe/nextboot [hunk 1]
```diff
@@ -17,7 +17,7 @@ from pxe.common import *
 import pxemngr.settings as settings
 
 if len(sys.argv) != 3:
-    error('Usage: %s <system name> <boot name>' % sys.argv[0])
+    error('Usage: pxemngr <system name> <boot name>')
```
Comment (fredericlepied) [inline pxe/nextboot]: you need to add nextboot like this:

`error('Usage: pxemngr nextboot <system name> <boot name>')
`

File: pxe/pxemngr [hunk 2]
```diff
@@ -27,7 +27,7 @@ print_subcommands() {
 }
 
 if [ $# -lt 1 ]; then
-    echo "Usage: `basename $0` <sub-command> [<args>...]" 1>&2
+    echo "Usage: pxemngr <sub-command> [<args>...]" 1>&2
```
Comment (fredericlepied) [inline pxe/pxemngr:30]: idem

File: tester/dpytest [hunk 3]
```diff
@@ -16,7 +16,7 @@ from pxe.common import *
 from tester.models import *
 
 if len(sys.argv) != 2:
-    error('Usage: %s <system name>|-l' % sys.argv[0])
+    error('Usage: pxemngr <system name>|-l')
```
Comment (fredericlepied) [inline tester/dpytest]: idem

File: tester/nexttest [hunk 4]
```diff
@@ -29,7 +29,7 @@ def set_next_test(system, name, abort=True):
     log.save()
 
 if len(sys.argv) < 3:
-    error('Usage: %s <system name> <test name> [<test name2>...]' % sys.argv[0])
+    error('Usage: pxemngr <system name> <test name> [<test name2>...]')
```
Comment (fredericlepied) [inline tester/nexttest]: idem

**Comment by fredericlepied** [comment]
Please fix according to my comments.

**Comment by ErwanAliasr1** [comment]
@fredericlepied good catch. Fixed & repushed.

**Comment by fredericlepied** [comment]
Thx Erwan!

**Pull request merged**